tests/utils.py

```diff
@@ -52,13 +51,6 @@ def get_pid_and_ray_node(a=0):
     return pid, node_id
 
 
-def get_random_str(length: int = 8):
```

Comment:
we'll also need to fix the import in `test_cluster.py` to import from runhouse utils instead of test utils